scripts/commit_classifier.py

```diff
@@ -315,10 +315,12 @@ def load_user(phid):
                 revision["fields"]["title"], revision["fields"]["summary"]
             )
 
+            latest_rev = repository.get_revs(hg, f"-{len(stack)}")
```

Comment:
We haven't applied the patches yet here, so this will return wrong revisions.

Reply:
The code should moved after the end of the for loop.

Reply:
Done!